src/models/base.py

```diff
@@ -87,6 +94,18 @@ def __init__(
         # by default, models which don't care will run on the CPU
         self.device = "cpu"
 
+    def _convert_delta_to_raw_values(
+        self, x: xr.Dataset, y: xr.Dataset, y_var: str, order: int = 1
+    ) -> xr.Dataset:
+        """When calculating the derivative we need to convert the change/delta
+        to the raw value for our prediction.
+        """
+        # x.shape == (pixels, featurespreds)
+        prev_ts = x[y_var].isel(time=-order)
```

Comment:
Should we take the negative of the passed order? Why not make it negative from the get go and pass the raw value?

Reply:
I think it is more interpretable remaining positive so that people know it is 'change since `n_timesteps` before'

Reply:
yea, that makes sense!